src/style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  box-sizing: border-box;
+}
+
+.body {
+  margin: 0;
+}
+
+.product_card {
+  font-family: Roboto, Arial, sans-serif;
+  width: 199px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  border-radius: 5px;
+}
+
+.card_img {
+  width: 160px;
+  height: 134px;
+  margin-left: 18px;
```

Comment:
better put elements to the center of the parent block.
use `text-align center`  or flex for the parent block